refinery/analysis_manager/utils.py

```diff
@@ -61,7 +61,7 @@ def create_nodeset_analysis(validated_analysis_config):
     :return: an Analysis instance
     :raises: RuntimeError
     """
-    custom_name = validated_analysis_config["custom_name"]
+    custom_name = validated_analysis_config.get("custom_name")
```

Comment:
In general, if we can avoid having None, it's a good thing. Here, we only use `custom_name` to set `temp_name`:
```python
if not custom_name:
         temp_name = current_workflow.name + " " + get_aware_local_time() \
             .strftime("%Y-%m-%d @ %H:%M:%S")
     else:
temp_name = custom_name
```

Could you just set `temp_name` directly, and use the optional second arg instead of the `if`? (One difference would be the handling of empty strings... not sure if that's a case we care about?)